grpcService/pkg/entities/requests.go

```diff
@@ -44,3 +44,15 @@ type DeleteUserRequest struct {
 type DeleteUserResponse struct {
 	Status Status
 }
+
+type UpdateUserRequest struct {
+	Name  string
+	Pass  string
```

Comment:
pass: an act or instance of moving past or through something. it won't hurt to write the full word, with larger scope better naming is required